docs/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Implement statement validation for Major Incident flaw (OSIDB-657)
 - Introduce new module for creating trackers in Jira (OSIDB-93)
 - Introduce aditional metadata in tasks generated from Taskman (OSIDB-861)
+- Implement adding new flaw comments (OSIDB-81)
```

Comment:
This should be in the `Unreleased` section.

Reply:
D'oh! Thanks for catching it.